tests/unit/amo/components/TestVPNPromoBanner.js

```diff
@@ -39,6 +46,12 @@ describe(__filename, () => {
     store = dispatchClientMetadata().store;
   });
 
+  // This is what would be stored in the experiment cookie for this experiment.
+  const experimentData = createExperimentData({
+    id: EXPERIMENT_CONFIG.id,
+    variantId: VARIANT_SHOW,
+  });
```

Comment:
I think we could introduce a function to create the cookies jar instead, given that the same snippet is used 3 times in this file:

```js
const createCookies = () => {
  return fakeCookies({
    get: sinon.stub().returns(createExperimentData({
      id: EXPERIMENT_CONFIG.id,
      variantId: VARIANT_SHOW,
    }))
  });
};
```